machineconfiguration/v1/types_pinnedimageset.go

```diff
@@ -0,0 +1,87 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=pinnedimagesets,scope=Cluster
+// +kubebuilder:subresource:status
```

Comment:
This must be removed since you no longer have status

Reply:
agreed